dpnp/backend/include/dpnp_iface.hpp

```diff
@@ -113,7 +113,7 @@ void dpnp_memory_memcpy_c(void* dst, const void* src, size_t size_in_bytes);
  * @param [out] result      Output array.
  * @param [in]  size        Number of input elements in `array`.
  */
-template <typename _DataType, typename _ResultType>
+template <typename _DataType>
```

Comment:
Leave it as it was. See, https://github.com/IntelPython/dpnp/pull/637#discussion_r586731916
The comment was for `any` but it suits also for `all`.